### PR TITLE
Core/Instance: may fix config option Rate.InstanceResetTime

### LINKED ISSUE #440: [CONF] CD reset does not work
**Description**:If you change the value to Rate.InstanceResetTime = 0.01
KD reset remains the same 6 days.
**Current behaviour**: Tell us what happens

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Open worldserver.conf
2. We replace: Rate.InstanceResetTime = 0.01
3. 

**Branch(es)**: 0.x / 1.x / master (Specify the branch(es) affected by this issue) 

**AC hash/commit**:  

**Operating system**:  
Debian 8
**Modules**:  Are you using modules? if yes, list them
no
**Other Customizations**: Are you using any core/db patches or other customizations? If yes please specify them here.

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing c++ create a Pull Request)

### PR DESCRIPTION
**Pull request by Viste**
**Changes proposed:**
-  Resotore back check for config in loading instance 

**Target branch(es):** master

**Issues addressed:** Closes #440 #284  #198 

**Tests performed:** (Does it build, tested in-game, etc)
only build all other plz test.  want community tests i'm no need this option 
**Known issues and TODO list:**
Need test

### CODE REVIEW

**Comment by Viste** [comment]
Test plz for me it shit no need.

File: src/game/Instances/InstanceSaveMgr.cpp [hunk 1]
```diff
@@ -278,7 +278,7 @@ void InstanceSaveManager::LoadResetTimes()
             continue;
 
         // the reset_delay must be at least one day
-        uint32 period = mapDiff->resetTime;
+        uint32 period = uint32(((mapDiff->resetTime * sWorld->getRate(RATE_INSTANCE_RESET_TIME))/DAY) * DAY);
```
Comment (Rochet2) [inline src/game/Instances/InstanceSaveMgr.cpp]: Whats going on here?
`((X * Y)/Z) * Z` = `X * Y` is it not?

Comment (Yehonal) [inline src/game/Instances/InstanceSaveMgr.cpp]: He just imported it from TC: https://github.com/TrinityCore/TrinityCore/blob/86b98686a95e23247ecb774fb23ecd5b8d94b97b/src/server/game/Instances/InstanceSaveMgr.cpp#L426

Comment (Viste) [inline src/game/Instances/InstanceSaveMgr.cpp]: ask it on trinitycore xD @Yehonal you a right

**Pull request merged**